<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KioskWindow</class>
 <widget class="QMainWindow" name="KioskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kiosk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>촬영</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>491</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
